GPU/Common/GPUDefGPUParameters.h

```diff
@@ -285,15 +285,15 @@
 #define GPUCA_LB_GPUTPCCFChargeMapFiller_resetMaps GPUCA_LB_CLUSTER_FINDER
 #define GPUCA_LB_GPUTPCCFChargeMapFiller_findFragmentStart GPUCA_LB_CLUSTER_FINDER
 #define GPUCA_LB_GPUTPCCFPeakFinder GPUCA_LB_CLUSTER_FINDER
-#define GPUCA_LB_GPUTPCCFNoiseSuppression_noiseSuppression GPUCA_LB_CLUSTER_FINDER
-#define GPUCA_LB_GPUTPCCFNoiseSuppression_updatePeaks GPUCA_LB_CLUSTER_FINDER
+#define GPUCA_LB_GPUTPCCFNoiseSuppression_step0 GPUCA_LB_CLUSTER_FINDER
+#define GPUCA_LB_GPUTPCCFNoiseSuppression_step1 GPUCA_LB_CLUSTER_FINDER
 #define GPUCA_LB_GPUTPCCFDeconvolution GPUCA_LB_CLUSTER_FINDER
 #define GPUCA_LB_GPUTPCCFClusterizer GPUCA_LB_CLUSTER_FINDER
-#define GPUCA_LB_GPUTPCCFStreamCompaction_nativeScanUpStart GPUCA_THREAD_COUNT_SCAN
-#define GPUCA_LB_GPUTPCCFStreamCompaction_nativeScanUp GPUCA_THREAD_COUNT_SCAN
-#define GPUCA_LB_GPUTPCCFStreamCompaction_nativeScanTop GPUCA_THREAD_COUNT_SCAN
-#define GPUCA_LB_GPUTPCCFStreamCompaction_nativeScanDown GPUCA_THREAD_COUNT_SCAN
-#define GPUCA_LB_GPUTPCCFStreamCompaction_compact GPUCA_THREAD_COUNT_SCAN
+#define GPUCA_LB_GPUTPCCFStreamCompaction_step0 GPUCA_THREAD_COUNT_SCAN
+#define GPUCA_LB_GPUTPCCFStreamCompaction_step1 GPUCA_THREAD_COUNT_SCAN
+#define GPUCA_LB_GPUTPCCFStreamCompaction_step2 GPUCA_THREAD_COUNT_SCAN
+#define GPUCA_LB_GPUTPCCFStreamCompaction_step3 GPUCA_THREAD_COUNT_SCAN
+#define GPUCA_LB_GPUTPCCFStreamCompaction_step4 GPUCA_THREAD_COUNT_SCAN
```

Comment:
Actually, why do you change this from the names to step[n] and then to integers below?
In my opinion, it was easier to read before.

Reply:
I thought it looked a bit cleaner. But it does hurt readability. I'll revert it.